### DISCUSSION TITLE
My deploy is failing and the logs dont appear

### DISCUSSION THREAD

**Discussion by gbrl-almeida** [vercel/community#1900] [Help] [answered]
### Summary

![image](https://user-images.githubusercontent.com/64490428/228269216-d1cd84d5-c0a5-4638-9ea1-db2e4cb978e1.png)

**Comment by amyegan** [comment, marked as answer]
Thanks for reaching out about this, Gabriel. The log delivery problem has been addressed and logs should be working normally now 🙇

**Reply by mkuhn0411** [reply]
I am having the same issue. Been trying since last night all deploys are timing out. What can I do to fix this?

**Reply by gbrl-almeida** [reply]
just redeploy. Seemed fixed here
